contrib/README.md

```diff
@@ -37,6 +37,7 @@ along with relevant permissions to the stitch-sdks bucket.
 
 ```bash
 lerna publish --force-publish="*"
+./update_browser_readme_version.sh
```

Comment:
[minor] this should go before `lerna publish`, otherwise the READMEs on npmjs.com will show an old version.